farmdata2_modules/fd2_tabs/fd2_barn_kit/transplantingReport.html

```diff
@@ -0,0 +1,639 @@
+<body>
+    <div id="seedingReport" v-cloak>
```

Comment:
The id should be updated here for the transplanting report.  It will then also need to be updated in the vue instance.